be/src/olap/bloom_filter.hpp

```diff
@@ -71,6 +70,16 @@ class BitSet {
         return true;
     }
 
+    bool init_with_deep_copy(uint64_t* data, uint32_t data_len) {
+        _data = new(std::nothrow) uint64_t[data_len];
```

Comment:
Who will delete allocated _data?

Reply:
Bitset will free the allocated _data in destructor